presto-docs/src/main/sphinx/develop/functions.rst

```diff
@@ -321,3 +321,65 @@ function follows:
   is used when performing a ``GROUP BY`` aggregation, and an implementation
   will be automatically generated for you, if you don't specify a
   ``AccumulatorStateFactory``
+
+
+Advanced Use Cases
+------------------
+
+Raw Block Inputs
+^^^^^^^^^^^^^^^^
+
+Both scalar and aggregation function annotations allow you to define methods
+which operate on native types. In Java, these native types are ``boolean``,
+``Slice``, and ``long``. For parameterized implementations or parametric types,
+the standard Java types can't be used as they can't fully represent the input
+data.
+
+To define a method handle which can accept *any* types use ``@BlockPosition`` in
```

Comment:
```suggestion
To define a method handle which can accept *any* types, use ``@BlockPosition`` in
```
Suggest a comma to separate the description ("define a method handle which can accept *any* types") from the how to (use ``@BlockPosition`` in conjunction with the ``@BlockIndex`` parameters.")